architecture/additional_concepts/flannel.adoc

```diff
@@ -15,7 +15,9 @@ toc::[]
 
 *flannel* is a virtual networking layer designed specifically for containers. 
 {product-title} can use it for networking containers instead of the default
-software-defined networking (SDN) components.
+software-defined networking (SDN) components. This is useful if running
+{product-title} within a cloud provider platform, such as OpenStack, and you
+want to avoid using dual Open vSwtich SDN on both platforms.
```

Comment:
typo "vSwtich"

oh, you just copied that out of configuring_sdn.adoc, where it is also misspelled...